api/src/main/java/com/netflix/iceberg/expressions/InclusiveManifestEvaluator.java

```diff
@@ -54,7 +54,20 @@ private ManifestEvalVisitor visitor() {
 
   public InclusiveManifestEvaluator(PartitionSpec spec, Expression rowFilter) {
     this.struct = spec.partitionType();
-    this.expr = Binder.bind(struct, rewriteNot(Projections.inclusive(spec).project(rowFilter)), true);
+    this.expr = Binder.bind(
+            struct,
+            rewriteNot(Projections.inclusive(spec, true).project(rowFilter)),
+            true
```

Comment:
Can this constructor then be rewritten as `this(spec, rowFilter, true)`?

Reply:
Good catch. Will fix.